src/js/tabs/osd.js

```diff
@@ -290,6 +223,205 @@ FONT.replaceLogoFromImage = function(img) {
     }
 };
 
+var LogoManager = LogoManager || {
```

Comment:
Considering that `osd.js` is already huge, and this is pretty much a standalone function, I think this should be extracted into a file of it's own.
Thoughts?

Reply:
Yes, I think this is something we maybe need to do more: the LogoManager is an independent functionality/API, so it will be great if extracted to a `logo_manager.js` (I'm not too sure in which folder, in the standard `src/js` maybe?) and simply use it from the `osd.js`.

Reply:
`src/js/` is the right location. And it should be `LogoManager.js` to match the object name.